std_msgs/Header header
float32[4] pos_d
uint32 type
float32 send_currentpos_freq
float32 send_desirepos_freq
float32 send_type_freq
uint32[5] flag